rotkehlchen/chain/ethereum/modules/aave/v1/decoder.py

```diff
@@ -104,7 +105,7 @@ def _decode_redeem_underlying_event(self, context: DecoderContext) -> DecodingOu
 
     def addresses_to_decoders(self) -> dict[ChecksumEvmAddress, tuple[Any, ...]]:
         return {
-            self.evm_inquirer.contracts.contract('AAVE_V1_LENDING_POOL').address: (self._decode_pool_event,),  # noqa: E501
+            self.evm_inquirer.contracts.contract(string_to_evm_address('0x398eC7346DcD622eDc5ae82352F02bE94C62d119')).address: (self._decode_pool_event,),  # noqa: E501
```

Comment:
This line is a bit silly since you are using the address to query for the address. Just leave it and add a comment saying that is the aave v1 lending pool